src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRate = 40 * days;
```

Comment:
You should declare a variable for price per day, according to checklist item 5.

Reply:
I mean you're looking at it, I don't understand what the issue is?

Reply:
I mean don't use value 40, use a variable instead.